### ISSUE TITLE
Display contents of a huge json row of a table as rows

### ISSUE THREAD

**Issue by cromartie1984** [yajra/laravel-datatables#2842]
### Summary of problem or feature request

Hi, I enjoy using your plugin, but is it possible to display contents of huge json (1000+ lines) row as multiple rows ?
Ex : My json row containing for instance :
`"409": {
            "date": "2017-04-04 00:00:00",
            "name": "cook"
        },
        "410": {
            "date": "2017-04-05 00:00:00",
            "name": "postman"
        }`
Displaying this into multiple rows and filtering it with options.
`<tr id="409"></tr><tr id="410"></tr>`
It would require to implement something like JSON_TABLE.
All I have been able to found is this (but it does not seems to work) : [https://stackoverflow.com/questions/59543251/what-is-the-equivalent-query-builder-expression-with-2-table-selects-in-laravel](url)

### System details

- Operating System
- PHP Version 8.0.21
- Laravel Version 8

**Comment by patricktorres27** [comment]
Hi @cromartie1984 

Are you trying to do something like this?
`
<tr>
    <tr id="409">....</tr>
    <tr id="410">....</tr>
</tr>
`

**Comment by cromartie1984** [comment]
> Hi @cromartie1984
> 
> Are you trying to do something like this?
> 
> ```
> <tr>
>     <tr id="409">....</tr>
>     <tr id="410">....</tr>
> </tr>
> ```

Yes in a way, want to display huge json data from a table row and display it like it was a full table.

**Comment by yajra** [comment]
You can use [addColumn](https://yajrabox.com/docs/laravel-datatables/master/add-column#closure) and then return a view. You can form the json_table markup there.

```php
-- DT
  ->addColumn('intro', function(User $user) {
    return view('json.table', compact('user'));
  })

-- json/table.blade.php

@foreach($user->json_table as $row)
<tr id="{{ $row['id'] }}">
  <td>{{ $row['key'] }}</td>
</tr>
@endforeach
```
